modules/mono/glue/GodotSharp/GodotSharp/Core/AABB.cs

```diff
@@ -157,7 +157,10 @@ public Vector3 GetEndpoint(int idx)
                 case 7:
                     return new Vector3(_position.x + _size.x, _position.y + _size.y, _position.z + _size.z);
                 default:
-                    throw new ArgumentOutOfRangeException(nameof(idx), String.Format("Index is {0}, but a value from 0 to 7 is expected.", idx));
+                {
+                    string text = String.Format("Index is {0}, but a value from 0 to 7 is expected.", idx);
```

Comment:
Should we use interpolated strings instead?
```suggestion
                    string text = $"Index is {idx}, but a value from 0 to 7 is expected.";
```